### ISSUE TITLE
Missing JSDoc description when it contains @param itself

### ISSUE THREAD

**Issue by kungfooman** [futurGH/ts-to-jsdoc#33]
Hey! I just tried to run ts-to-jsdoc on itself and came across this:

![image](https://github.com/futurGH/ts-to-jsdoc/assets/5236548/c94fa6ce-b6ee-45ce-b37d-40f10a5380e9)

(it cuts off the description when encountering `@param`)

Another case:

![image](https://github.com/futurGH/ts-to-jsdoc/assets/5236548/4d954388-1a20-46f8-a997-533dfe8ffaef)

**Comment by futurGH** [comment]
Ooh, good catch! The compiler does seem to interpret those as actual JSDoc tags so I'll take that as user error on my part :^)

**Comment by kungfooman** [comment]
You mean it's a bug in ts-morph? TS compiler handles it correctly in a little test

**Comment by futurGH** [comment]
Take a look at [this](https://ts-ast-viewer.com/#code/PQKhCgAIUgXBTAzrSABADgQwE6YLaRa4GwAWAlopJdQHYLbrbwIAmkmVR+L82krAPYBjAK5549TLHKDaUEMHAAzUbWEy5cJLAAU3PAC5IybOVoBzAJSQA3gF8gA). If you select the FunctionDeclaration and run `node.jsDoc[0]` in your browser console, you'll see that "test" is interpreted as the function's description, while everything following `@param` is interpreted as documenting the parameter.

In other words, it's not a bug — to the compiler, the function description is just "Generate".

**Comment by kungfooman** [comment]
Interesting, I only had a quick look at Intellisense before:

![image](https://github.com/futurGH/ts-to-jsdoc/assets/5236548/0d8f5004-5df4-4db0-b44c-dd6e98b79d51)

(it correctly attaches everything as a description only, no description for `param`)

Seems like it doesn't just rely on the TypeScript AST, at least a certain level of extra code - I couldn't figure out where exactly the extra handling happens :thinking:
